docs/en/integrations/deepnote.md

```diff
@@ -0,0 +1,40 @@
+---
+sidebar_label:  Deepnote
+sidebar_position: 11
+keywords: [clickhouse, Deepnote, connect, integrate, notebook]
+description: Efficiently query very large datasets, analyzing and modeling in the comfort of known notebook environment.
+---
+
+# Connect ClickHouse to Deepnote
+
+<a href="https://www.deepnote.com/" target="_blank">Deepnote</a> is a collaborative data notebook built for teams to discover and share insights. In addition to being Jupyter-compatible, it works magically in the cloud and provides you with one central place to collaborate and work on data science projects efficiently.
```

Comment:
```suggestion
<a href="https://www.deepnote.com/" target="_blank">Deepnote</a> is a collaborative data notebook built for teams to discover and share insights. In addition to being Jupyter-compatible, it works in the cloud and provides you with one central place to collaborate and work on data science projects efficiently.
```

Reply:
I see that "magically" comes from the Deepnote website, but can we leave the word out or replace it with the key cloud feature that differentiates Deepnote?

Reply:
Sure - removed it